lessons/requirements.md

```diff
@@ -1,24 +1,26 @@
 ## Requirements
```

Comment:
are these the requirements of all events codelabs? how about creating a common requirements template for later use?

Reply:
strongly agree we have one requirement including how to get project firefly and console, the only difference is to get event webhook